src/routes/local.ts

```diff
@@ -1,13 +1,46 @@
 import express from 'express'
 import n2yo from './../api/n2yo/index.js'
+import SatelliteUtilities from '../utils/satelliteUtils'
+import _ from 'lodash'
 
 const router = express.Router()
+const propsToPick = ['_id', 'officialName', 'perigee', 'apogee', 'operator', 'operatorCountry', 'purpose']
 
 router.get('/two-line-elements/:id', (req, res) => {
-    if (req.params.id == undefined) return res.sendStatus(400)
+    if (req.params.id === undefined) return res.sendStatus(400)
     n2yo.getTwoLineElements(req.params.id)
         .then(result => res.json(result))
         .catch(() => res.sendStatus(400))
 })
 
+router.get('/satellite', (req, res) => {
+    if (req.body.id === undefined && req.body.name === undefined) return res.sendStatus(400)
```

Comment:
If you'll receive `NaN`, `null`, `0` or `''` as `.name` or `.id`, you will end up with pending http request, never responding to it.

Reply:
Also, you use lodash, so you can use `_.isUndefined(x)`. It won't fix the issues i've pointed out, it's more like a tip for the future.